router.php

```diff
@@ -11,6 +11,8 @@
 	require 'Models/' . str_replace('\\', '/', $className) . '.php';
 });
 
+WebApp\WebApp::requireHTTPS();//force redirect to https
```

Comment:
Maybe it should be better to move the code from the function "requireHTTPS" directly to the router.
Also we can check if the PHP built-in server is used in the same if condition:

```
if (!WebApp::isSecure() && php_sapi_name() != 'cli-server')
{
  header('Location: https://' . $_SERVER['HTTP_HOST'] . $_SERVER['REQUEST_URI'], TRUE, 301);
  exit;
}
```

Like that we can continue testing normally with the dev environment 🙂

Reply:
Lemme update that